lib/flutter_linkify.dart

```diff
@@ -17,6 +17,9 @@ export 'package:linkify/linkify.dart'
 /// Callback clicked link
 typedef LinkCallback(LinkableElement link);
 
+/// Function for dynamic display text of link
+typedef String DisplayTextFn(String s);
```

Comment:
```suggestion
typedef String ElementFormatter(LinkifyElement element);
```

Let's make it more generic